integration_tests/src/main/python/delta_lake_liquid_clustering_test.py

```diff
@@ -385,7 +386,8 @@ def merge_table(spark, path):
             WHEN NOT MATCHED THEN
               INSERT *
             """
-        spark.sql(sql).show()
+        print(f"Running sql on {type} table:\n{sql}")
+        spark.sql(sql).collect()
```

Comment:
Why was this change needed? How was it working before AFAIK `show()` doesn't return anything